### PR TITLE
[DECLUTTER] [USING] Explain Wasabi like Im 5

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch introduces Wasabi with a very short `explain it like I'm 5` section. It covers the download, installation, start, wallet generation, receiving, sending, and coinjoining very briefly. It also links to the main chapters for further details. The starting page call to action button `Reclaim Your Privacy` now links to this page.

### CODE REVIEW

File: docs/using-wasabi/ELI5.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "title": "Explain Wasabi like I'm 5",
+  "description": "Explaining very simply how to install and start Wasabi, as well as receiving, sending and coinjoining. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
+}
+---
+
+# Explain Wasabi like I'm 5
+
+[[toc]]
+
+## Introduction
```
Comment (PulpCattel) [inline docs/using-wasabi/ELI5.md]: Can I ask why you didn't use [this](https://docs.wasabiwallet.io/FAQ/FAQ-Introduction.html#explain-wasabi-like-i-m-5)?

File: docs/using-wasabi/ELI5.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+You can then install Wasabi as you would do any other software on your computer.
+
+See [this chapter](/using-wasabi/InstallPackage.md) for a detailed step-by-step tutorial for all operating systems, and also how to verify the PGP signature.
+
+## Generating a Wallet
+
+When for the first time you start Wasabi, it will open in the tab to generate a new wallet.
+You can set a unique name for the wallet so that you remember what it is for.
+Choose a very secure password, as this encrypts the secrets, and should not be guessed by others.
+You will always need this password when you want to spend your bitcoins, so take great care to back it up properly.
```
Comment (PulpCattel) [inline docs/using-wasabi/ELI5.md]:
```suggestion
You will always need this password when you want to spend your bitcoins (even outside of Wasabi), so take great care to back it up properly.
```
Although this actually says it `You will always need this password when you want to spend your bitcoins`, I'm not sure a newbie will catch that this extends outside of Wasabi.

File: docs/using-wasabi/ELI5.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
+Go to the `CoinJoin` tab, select the coins you want to make private, type in your password and click enqueue.
+Now have a little patience, within a short while you should have sufficient anonymity set to protect your privacy.
+
+See [this chapter](/using-wasabi/CoinJoin.md) for a thorough analysis of CoinJoin, details on the Wasabi implementation and best practices how to use it properly.
+
+## Sending bitcoin
+
+In the Wasabi `Send` tab you can give your bitcoins to someone else, or send them to another wallet of yours.
+Select the coins which you want to spend, paste the address of the receiver, label the observers who know of this transaction, specify the amount the receiver should get and select how fast you want the transaction to confirm.
```
Comment (PulpCattel) [inline docs/using-wasabi/ELI5.md]:
```suggestion
* Select the coins which you want to spend.
* Insert the address of the receiver.
* Label the observers who know of this transaction.
* Specify the amount the receiver should get.
* Set the fee to select how fast you want the transaction to confirm.
```
What about a list here instead?

File: docs/using-wasabi/ELI5.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
+Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
+Go to the `CoinJoin` tab, select the coins you want to make private, type in your password and click enqueue.
+Now have a little patience, within a short while you should have sufficient anonymity set to protect your privacy.
+
+See [this chapter](/using-wasabi/CoinJoin.md) for a thorough analysis of CoinJoin, details on the Wasabi implementation and best practices how to use it properly.
+
+## Sending bitcoin
+
+In the Wasabi `Send` tab you can give your bitcoins to someone else, or send them to another wallet of yours.
```
Comment (PulpCattel) [inline docs/using-wasabi/ELI5.md]:
```suggestion
In the Wasabi `Send` tab you can give your bitcoins to someone else, or send them to another wallet of yours.
You'll have to:
```
If you like the list.

File: docs/using-wasabi/ELI5.md (hunk 1 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/ELI5.md]: You mean the `[[toc]]`? It is an automated vuepress function to build a table of content of the headers in the markdown file, always up to date and with proper links. It is used in every chapter, and results in this...
screenshot_20200206_143346

File: docs/using-wasabi/ELI5.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/ELI5.md]: Yes, it also means that both the recovery words and digital wallet file are encrypted with the password, and I'm not sure if that is conveyed properly either...

Though I don't fancy the `(even outside of Wasabi)`, does anyone know a more beautiful way of describing it?

File: docs/using-wasabi/ELI5.md (hunk 1 above)
Comment (PulpCattel) [reply docs/using-wasabi/ELI5.md]: No, sorry I wasn't clear. There was already an ELI5 introduction to Wasabi in the documentation FAQ.
https://docs.wasabiwallet.io/FAQ/FAQ-Introduction.html#explain-wasabi-like-i-m-5
I was curious if there were reasons to rewrite it.

File: docs/using-wasabi/ELI5.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+You'll have to:
+* Select the coins which you want to spend.
+* Insert the address of the receiver.
+* Label the observers who know of this transaction.
+* Specify the amount the receiver should get.
+* Set the fee to select how fast you want the transaction to confirm.
```
Comment (PulpCattel) [inline docs/using-wasabi/ELI5.md]:
```suggestion
* Set the fee to select how fast you want the transaction to confirm.

```
Forgot space

File: docs/using-wasabi/ELI5.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+Installing Wasabi is super easy.
+Go to the official website [wasabiwallet.io](https://wasabiwallet.io), scroll down and download the version for your operating system.
+You can then install Wasabi as you would do any other software on your computer.
+
+See [this chapter](/using-wasabi/InstallPackage.md) for a detailed step-by-step tutorial for all operating systems, and also how to verify the PGP signature.
+
+## Generating a Wallet
+
+When for the first time you start Wasabi, it will open in the tab to generate a new wallet.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
When you start Wasabi for the first time, the tab to generate a new wallet will be open.
```

File: docs/using-wasabi/ELI5.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+You can set a unique name for the wallet so that you remember what it is for.
+Choose a very secure password, as this encrypts the secrets, and should not be guessed by others.
+You will always need this password when you want to spend your bitcoins, so take great care to back it up properly.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]: You will always need this password in order to spend your bitcoins, so take great care to back it up properly.
__Important:__ If you ever need to restore this wallet on Wasabi, or any other wallet, you will need both: this password  __AND__ your 12 recovery words.

File: docs/using-wasabi/ELI5.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+In the next step, Wasabi will show your 12 recovery words, these must be carefully backed up also, as they can help you access your bitcoins when your computer breaks.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
In the next step, Wasabi will show your 12 recovery words.
These must be carefully backed up also, as they will be needed to access your bitcoins when your computer breaks.
Make sure you store the backup of your recovery words in a different location than the backup of your password.
```

File: docs/using-wasabi/ELI5.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+See [this chapter](/using-wasabi/WalletGeneration.md) for more information on how to securely generate your wallet.
+
+## Receiving bitcoin
+
+In order to receive your first bitcoins, you can generate an address in the `Receive` tab.
+But before you must label it with the observer who knows that this address is yours.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
But first, you must label it with the names of any observers who know that this address is yours.
```

File: docs/using-wasabi/ELI5.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+This is an important feature so that you know which coin is from whom, and it will help with your privacy in the future.
+
+See [this chapter](/using-wasabi/Receive.md) for a tutorial on how to properly label your addresses and receive bitcoin.
+
+## CoinJoin with Wasabi
+
+If you don't like that your employer knows about where you spend your money, or that a merchant can find out how much money you have, then you should CoinJoin your bitcoin, as this breaks the transaction history.
+Although the CoinJoin protocol is a bit complex, it is very easy to use in Wasabi.
+Go to the `CoinJoin` tab, select the coins you want to make private, type in your password and click enqueue.
+Now have a little patience, within a short while you should have sufficient anonymity set to protect your privacy.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
Now have a little patience, within a short while you should have a sufficient anonymity set to protect your privacy.
```

File: docs/using-wasabi/ELI5.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
+---
+{
+  "title": "Explain Wasabi like I'm 5",
+  "description": "Explaining very simply how to install and start Wasabi, as well as receiving, sending and coinjoining. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
  "description": "A very simple explanation of how to install and start Wasabi, as well as receiving, sending, and CoinJoin-ing. 
This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

File: docs/using-wasabi/ELI5.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+# Explain Wasabi like I'm 5
+
+[[toc]]
+
+## Introduction
+
+Wasabi is a Bitcoin wallet, tailor-made to protect your privacy.
+It is a piece of software that runs on your computer and helps you to manage your bitcoins.
+You can generate a new set of private keys, or import already existing ones.
+With Wasabi you can receive bitcoins to your full control, and you can send them to someone else without permission.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
With Wasabi you can receive bitcoins into your full control, and you can send them to someone else without needing permission from any other person or entity.
```

File: docs/using-wasabi/ELI5.md (hunk 11 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/ELI5.md]:
```suggestion
  "description": "A very simple explanation of how to install and start Wasabi, as well as receiving, sending, and coinjoining. 
This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

File: docs/using-wasabi/ELI5.md (hunk 1 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/ELI5.md]: ah, this is part of the decluttering effort. https://github.com/zkSNACKs/WasabiDoc/issues/537

File: docs/using-wasabi/ELI5.md (hunk 7 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/ELI5.md]:
```suggestion
You will always need this password in order to spend your bitcoins, so take great care to back it up properly.
Important: If you ever need to restore this wallet on Wasabi, or any other wallet, you will need both: this password AND your 12 recovery words.```

File: docs/using-wasabi/ELI5.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+See [this chapter](/using-wasabi/InstallPackage.md) for a detailed step-by-step tutorial for all operating systems, and also how to verify the PGP signature.
+
+## Generating a Wallet
+
+When you start Wasabi for the first time, the tab to generate a new wallet will be open.
+You can set a unique name for the wallet so that you remember what it is for.
+Choose a very secure password, as this encrypts the secrets, and should not be guessed by others.
+You will always need this password in order to spend your bitcoins, so take great care to back it up properly.
+Important: If you ever need to restore this wallet on Wasabi, or any other wallet, you will need both: this password AND your 12 recovery words.```
```
Comment (PulpCattel) [inline docs/using-wasabi/ELI5.md]:
```suggestion

:::warning
If you ever need to restore this wallet on Wasabi, or any other wallet, you will need both: this password AND your 12 recovery words.
:::

```
Fix and add warning highlight.

File: docs/using-wasabi/ELI5.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+}
+---
+
+# Explain Wasabi like I'm 5
+
+[[toc]]
+
+## Introduction
+
+Wasabi is a Bitcoin wallet, tailor-made to protect your privacy.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/ELI5.md]:
```suggestion
Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet.
```

File: docs/using-wasabi/ELI5.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+It is a piece of software that runs on your computer and helps you to manage your bitcoins.
+You can generate a new set of private keys, or import already existing ones.
+With Wasabi you can receive bitcoins into your full control, and you can send them to someone else without needing permission from any other person or entity.
+There is also a powerful CoinJoin feature to help obfuscate your transaction history.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/ELI5.md]:
```suggestion
There is also a powerful [CoinJoin](/using-wasabi/CoinJoin.md) feature to help obfuscate your transaction history.
```

**Comment by MaxHillebrand** [comment]
I have now removed the FAQ and ported some sentences of it into this chapter.
Although I have merged @RiccardoMasutti suggestion, I have now again removed the link to CoinJoin in the introduction, as we would have to link to all the chapters [generate, receive, send, ... ] too, and that would clutter this first paragraph. All the links are in the standalone paragraphs following, so I don't think this is missing anything.

The introduction mentions full node and hardware wallet, though I don't think there should be a standalone paragraph for these, as this is already a bit advanced.

Please give it another round of review now, and ACK/NACK for merge, I think it is ready.

File: docs/using-wasabi/ELI5.md [hunk 16]
```diff
@@ -0,0 +1,83 @@
+---
+{
+  "title": "Explain Wasabi like I'm 5",
+  "description": "A very simple explanation of how to install and start Wasabi, as well as receiving, sending, and coinjoining.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
  "description": "A very simple explanation of how to install and start Wasabi, as well as receiving, sending, and CoinJoining. 
```

File: docs/using-wasabi/ELI5.md [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+## Introduction
+
+Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet.
+It is a piece of software that runs on your computer and helps you to manage your bitcoins.
+Although Wasabi has some very advanced magic under the hood, it is rather easy to use.
+You can generate a new set of private keys, or import already existing ones.
+With Wasabi you can receive bitcoins into your full control, and you can send them to someone else without needing permission from any other person or entity.
+There is also a powerful CoinJoin feature to help obfuscate your transaction history.
+You can use Wasabi to manage your hardware wallet, and it even connects to your own full node.
+Of course, Wasabi is libre and open source, which means you have full control over the software you manage your money with.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
Of course, Wasabi is libre and open-source, which means you have full control over the software you manage your money with.
```

File: docs/using-wasabi/ELI5.md [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+
+## Installing Wasabi
+
+Installing Wasabi is super easy.
```
Comment (davterra) [inline docs/using-wasabi/ELI5.md]:
```suggestion
Installing Wasabi is super-easy.
```
